### PR TITLE
feat: display the number of queries in the past 7 days on each indexer card in the dashboard

### LINKED ISSUE #795: Display the number of queries in the past 7 days on each indexer card in the dashboard

### PR DESCRIPTION
**Pull request by Kevin101Zhang**
BOS component indexercard now queries past 7 day query count.

### CODE REVIEW

**Comment by Kevin101Zhang** [comment]
Screenshot 2024-06-25 at 4 49 05 PM

File: frontend/widgets/src/QueryApi.IndexerExplorer.jsx [hunk 1]
```diff
@@ -1,46 +1,49 @@
...
-  });
-}
+const santizedAccountId = accountId.replaceAll(".", "_");
+
+const [selectedTab, setSelectedTab] = useState(props.tab && props.tab !== "all" ? props.tab : "all");
+const [myIndexers, setMyIndexers] = useState([]);
+const [allIndexers, setAllIndexers] = useState([]);
+
+const fetchIndexerData = () => {
+  const url = 'https://storage.googleapis.com/databricks-near-query-runner/output/query-api-usage/indexers.json';
```
Comment (morgsmccauley) [inline frontend/widgets/src/QueryApi.IndexerExplorer.jsx:9]: Probably need to make this an environment variable, and use the dev endpoint when it's ready

Comment (Kevin101Zhang) [reply frontend/widgets/src/QueryApi.IndexerExplorer.jsx]: done

File: frontend/widgets/src/QueryApi.IndexerCard.jsx [hunk 2]
```diff
@@ -1,11 +1,9 @@
-const accountId = props.accountId || context.accountId;
-const indexerName = props.indexerName;
+const { accountId, indexerName, lastDeploymentDate, numDeployements, numQueries, originalDeploymentDate } = props;
```
Comment (eduohe) [inline frontend/widgets/src/QueryApi.IndexerCard.jsx:1]: I think we need to show `originalDeploymentDate` in the card too

**Review by eduohe — DISMISSED** [review]

File: frontend/widgets/src/QueryApi.IndexerCard.jsx (hunk 2 above)
Comment (Kevin101Zhang) [reply frontend/widgets/src/QueryApi.IndexerCard.jsx:1]: I belive this is more for internal use. Clarified with pavel

**Pull request merged**